fbw-a32nx/src/systems/fmgc/src/efis/EfisSymbols.ts

```diff
@@ -813,8 +813,8 @@ export class EfisSymbols<T extends number> {
           ident: fixInfo.fix.ident,
           location: fixInfo.fix.location,
           type: NdSymbolTypeFlags.FixInfo,
-          radials: fixInfo.radials.map((it) => it.trueBearing),
-          radii: fixInfo.radii.map((it) => it.radius),
+          radials: fixInfo?.radials.map((it) => it.trueBearing),
+          radii: fixInfo?.radii.map((it) => it.radius),
```

Comment:
fixInfo cannot be nullish here.
```suggestion
          radials: fixInfo.radials.map((it) => it.trueBearing),
          radii: fixInfo.radii.map((it) => it.radius),
```

Reply:
Fixed